pkg/daemon/controller_linux.go

```diff
@@ -196,7 +196,7 @@ func (c *Controller) reconcileRouters(event *subnetEvent) error {
 	joinCIDR := make([]string, 0, 2)
 	cidrs := make([]string, 0, len(subnets)*2)
 	for _, subnet := range subnets {
-		if (subnet.Spec.Vlan != "" && !subnet.Spec.LogicalGateway) || subnet.Spec.Vpc != c.config.ClusterRouter || !subnet.Status.IsReady() {
+		if subnet.Spec.Vpc != c.config.ClusterRouter || !subnet.Status.IsReady() {
```

Comment:
Why is `subnet.Spec.Vlan != "" && !subnet.Spec.LogicalGateway` deleted?

Reply:
I will recover this， this could filter cidr before later diff routes